build.gradle

```diff
@@ -277,20 +278,21 @@ dependencies {
   // https://mvnrepository.com/artifact/org.springframework.retry/spring-retry
   implementation group: 'org.springframework.retry', name: 'spring-retry', version: '1.3.4'
 
-  implementation group: 'io.springfox', name: 'springfox-swagger2', version: '2.9.2'
-  implementation group: 'org.projectlombok', name: 'lombok', version: '1.18.22'
-  implementation group: 'io.rest-assured', name: 'rest-assured', version:  '4.3.1'
-  implementation group: 'io.rest-assured', name: 'json-path', version: '4.3.1'
-  implementation group: 'io.rest-assured', name: 'xml-path', version: '4.3.1'
-  implementation group: 'org.elasticsearch', name: 'elasticsearch', version: '7.16.3'
+  implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.6.12'
```

Comment:
Question -- I'm guessing this is required by latest Spring upgrades to migrate from springfox to springdoc dependency, if so this is fine.

Reply:
It was written in the ticket to do so, and I have found out that fox is no longer supported, whereas springdoc is still maintained